pylintrc

```diff
@@ -0,0 +1,272 @@
+# -*- coding: utf-8; mode: conf -*-
+# lint Python modules using external checkers.
```

Comment:
I'm not super familiar with pylint, but is it possible for this to be a `.pylintrc` rather than a visible file in the top-level source directory?

Reply:
I can rename the file / no problem.